mlx_graphs/datasets/dataset.py

```diff
@@ -1,16 +1,21 @@
 import os
 from abc import ABC, abstractmethod
+from pathlib import Path
 from typing import Optional
 
 from mlx_graphs.data import GraphData
 
-DEFAULT_BASE_DIR = "~/.mlx_graphs_data/"
+# Default path for downloaded datasets is the root of mlx-graphs package
+file_dir = os.path.dirname(os.path.realpath(__file__))
+DEFAULT_BASE_DIR = os.path.join(
+    Path(file_dir).parents[1].absolute(), ".mlx_graphs_data/"
+)
```

Comment:
Rethinking about it - shall we put it in the same folder where the code is being run? So that it's easier for anyone to explore it (rather than being in the mlx-graphs package repo inside a venv)

Reply:
Ok I just modified